### PR TITLE
Learning Team Agenda - October 29th, 2020

### PR DESCRIPTION
**Pull request by ijlee2**
Please add comments or make edits if you want to add to the agenda! Core team and community members are all welcome to participate and add discussion points.

### CODE REVIEW

File: learning-team/2020-10/2020-10-29.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+See: [Initiatives board](https://github.com/orgs/ember-learn/projects/33)
+
+### Review PR queue
+See: [tracking sheet](https://docs.google.com/spreadsheets/d/1sPyN9z9wZMpTNwqCfa6R9QSPZkIW4iQd-H4gZC7ILLk/edit#gid=2035777454)
+
+## Topics
+
+<!-- If you would like to add a topic to the agenda please add a suggestion to the PR using the following format: -->
+<!-- ### Your topic (INITIALS, expected duration in minutes) -->
+<!-- replace with topic -->
```
Comment (ijlee2) [inline learning-team/2020-10/2020-10-29.md:39]:
```suggestion
### Follow-up to Ember Language Server (ELS) discussion (IL, 15 min)

On October 28, I met with Aleksandr (`@lifeart` on Discord), the main maintainer of Unstable ELS, to address possible concerns and discuss a possible path for the future. Let's catch up and discuss how we can help with logistics.
```

File: learning-team/2020-10/2020-10-29.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+See: [tracking sheet](https://docs.google.com/spreadsheets/d/1sPyN9z9wZMpTNwqCfa6R9QSPZkIW4iQd-H4gZC7ILLk/edit#gid=2035777454)
+
+## Topics
+
+<!-- If you would like to add a topic to the agenda please add a suggestion to the PR using the following format: -->
+<!-- ### Your topic (INITIALS, expected duration in minutes) -->
+### Follow-up to Ember Language Server (ELS) discussion (IL, 15 min)
+
+On October 28, I met with Aleksandr (`@lifeart` on Discord), the main maintainer of Unstable ELS, to address possible concerns and discuss a possible path for the future. Let's catch up and discuss how we can help with logistics.
+<!-- replace with topic -->
```
Comment (ijlee2) [inline learning-team/2020-10/2020-10-29.md:42]: Placeholder for @locks 
```suggestion
### Discuss 2 RFCs (RM, ?? min)

<!-- Provide description -->

- https://emberjs.github.io/rfcs/0617-rfc-stages.html
- https://emberjs.github.io/rfcs/0649-deprecation-staging.html
```

File: learning-team/2020-10/2020-10-29.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+<!-- replace with topic -->
```
Comment (ijlee2) [inline learning-team/2020-10/2020-10-29.md:43]: Placeholder for @mansona / @locks (regarding upgrade.emberjs.com)

```suggestion
### Review subdomains (CM, ?? min)

<!-- Provide description -->
```

Comment (mansona) [reply learning-team/2020-10/2020-10-29.md:54]: I added this directly to the file 👍

**Pull request merged**